crates/ra_ide_api/src/references/search_scope.rs

```diff
@@ -0,0 +1,99 @@
+//! FIXME: write short doc here
+
+use std::collections::HashSet;
```

Comment:
```suggestion
use rustc_hash::FxHashSet;
```

It's basically std::collections::HashSet, but with a faster hash function